index.html

```diff
@@ -242,6 +244,52 @@ <h4>Skills</h4>
                 </div>
             </div>
         </section>
+        <!--Contact form section-->
+        <section class="contact-me">
+            <!--contact me section main text-->
+            <div class="contactme-text">
+                <h1 class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            </div>
+            <!--form div-->
+            <div class="form-outline">
+                <form action="https://formspree.io/f/xeqwblvq" method="POST">
+                  <ul class="contactme-ul">
+                    <li class="name">
+                        <label for="name"></label>
+                        <input type="text" id="name" name="user-name" maxlength="30" placeholder="Full Name" required/>
+                    </li>
+                    <li class="email">
+                        <label for="mail"></label>
+                        <input type="email" id="mail" name="user-email" placeholder="Email address" required/>
+                    </li>
+                    <li class="message">
+                        <label for="message"></label>
+                        <textarea id="message" name="user-message" maxlength="500" placeholder="Hey I'd like to connect to..." required></textarea>
+                    </li>
+                    <li class="submit-button">
+                        <label for="submitbutton"></label>
```

Comment:
- [ ] Please remove the **label** because it is meant for adding `placeholder` to make the user aware regarding what kind of information you need to pass in the input field, but you already have the *placeholder* attribute in the `input` tag so that's why you don't need it anymore.